sympy/physics/vector/vector.py

```diff
@@ -678,7 +678,17 @@ def subs(self, *args, **kwargs):
         return Vector(d)
 
     def magnitude(self):
-        """Returns the magnitude (Euclidean norm) of self."""
+        """Returns the magnitude (Euclidean norm) of self.
+
+        Warnings
+        ========
```

Comment:
I think the heading "Warnings" is supposed to explain actual warnings that are returned. Just make this "Notes".

Reply:
The `Warns` section is for actual warnings, `Warnings` is the one for this.

Reply:
I'm wrong about that:

https://numpydoc.readthedocs.io/en/latest/format.html#warns